app/src/main/java/android/example/wordlehelper/StatsManager.kt

```diff
@@ -20,10 +28,60 @@ class StatsManager(context: Context) {
         return Gson().fromJson(userStatsString, UserStats::class.java)
     }
 
-    fun storeStats(data: UserStats) {
+    private fun storeStats(data: UserStats) {
         val userStatsJson: String = Gson().toJson(data)
         file.writeText(userStatsJson)
     }
+
+     fun onGiveUpUpdateStats() {
```

Comment:
I wouldn't call this method like this. The stats manager is a class to receive orders, so all the methods should be named as direct actions, In this case, _storeGiveUpStats_.

Reply:
The verb should be the first word, so I suggest to change all other methods.